.release/ci.hcl

```
@@ -0,0 +1,216 @@
+schema = "1"
+
+project "packer" {
+  team = "#team-packer"
+  slack {
+    notification_channel = "C032JM64THT"
```

This channel ID is incorrect
```suggestion
    notification_channel = "C04SJ33GY"
```

---

I'm using this for testing :) I'll update it to #proj-packer-releases before merging.